### PR TITLE
National Minimum Wage 1 April 2016

### PR DESCRIPTION
**Pull request by pmanrubia**
Trello card: https://trello.com/c/kZ0I4a4G/58-national-minimum-wage-april-2016-update

Covers the updates required after 1 April 2016:
- If over 16 but under 24 and not an apprentice, use the national minimum wage calculations
- If over 25 use the national living wage rates

# Factchecks
* https://smart-answers-pr-2423.herokuapp.com/am-i-getting-minimum-wage
* https://smart-answers-pr-2423.herokuapp.com/minimum-wage-calculator-employers

# Expected Changes

* https://www.gov.uk/am-i-getting-minimum-wage
* https://www.gov.uk/minimum-wage-calculator-employers

A detailed detailed description of the expected changes has been described in this document by the Content Digital Team: https://docs.google.com/document/d/1HYr7oPtx1zyGo1uiq4-dlqljsfVzR7YKRcc0zhvRksw/edit

# Employee

## Before

* Over 25 && Below National Living Wage
https://gov.uk/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/25/1/8.0/40.0/0.0/no

* Over 25 && Over National Living Wage
https://gov.uk/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/25/1/8.0/640.0/0.0/no

* Below 25 && Below National Minimum Wage
https://gov.uk/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/23/1/8.0/40.0/0.0/no

* Below 25 && Over National Minimum Wage
https://gov.uk/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/23/1/8.0/640.0/0.0/no

## After

* Over 25 && Below National Living Wage
https://smart-answers-pr-2423.herokuapp.com/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/25/1/8.0/40.0/0.0/no

* Over 25 && Over National Living Wage
https://smart-answers-pr-2423.herokuapp.com/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/25/1/8.0/640.0/0.0/no

* Below 25 && Below National Minimum Wage
https://smart-answers-pr-2423.herokuapp.com/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/23/1/8.0/40.0/0.0/no

* Below 25 && Over National Minimum Wage
https://smart-answers-pr-2423.herokuapp.com/am-i-getting-minimum-wage/y/current_payment/not_an_apprentice/23/1/8.0/640.0/0.0/no

# Employers

## Before

* Below 25 && Below National Minimum Wage
https://www.gov.uk/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/22/1/8.0/48.0/0.0/no

* Below 25 && Over National Minimum Wage
https://www.gov.uk/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/22/1/8.0/648.0/0.0/no

* Over 25 && Below National Living Wage
https://www.gov.uk/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/25/1/8.0/40.0/0.0/no

* Over 25 && Over National Living Wage
https://www.gov.uk/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/25/1/8.0/640.0/0.0/no

## After

* Below 25 && Below National Minimum Wage
https://smart-answers-pr-2423.herokuapp.com/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/22/1/8.0/48.0/0.0/no

* Below 25 && Over National Minimum Wage
https://smart-answers-pr-2423.herokuapp.com/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/22/1/8.0/648.0/0.0/no

* Over 25 && Below National Living Wage
https://smart-answers-pr-2423.herokuapp.com/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/25/1/8.0/40.0/0.0/no

* Over 25 && Over National Living Wage
https://smart-answers-pr-2423.herokuapp.com/minimum-wage-calculator-employers/y/current_payment/not_an_apprentice/25/1/8.0/640.0/0.0/no

### CODE REVIEW

File: lib/smart_answer_flows/am-i-getting-minimum-wage/outcomes/current_payment_below.govspeak.erb [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+  <% else %>
+    The National Minimum Wage per hour for your age | Your actual pay | For each pay period your employer owes you
+    - | - | -
+    <%= format_money(calculator.minimum_hourly_rate) %> | <%= format_money(calculator.total_hourly_rate) %> * | <%= format_money(calculator.total_underpayment) %>
+  <% end %>
 
-  \* This figure has been rounded to the nearest penny.
-
-  ^Your actual pay should increase to at least £7.20 per hour from 1 April 2016.^
+    \* This figure has been rounded to the nearest penny.
```
Comment (floehopper) [inline lib/smart_answer_flows/am-i-getting-minimum-wage/outcomes/current_payment_below.govspeak.erb]: Very minor: The indentation of this line looks wrong.

**Comment by floehopper** [comment]
In general this looks like a big improvement on #2419 - I found it much easier to follow in terms of the structure of the commits.

As I mention in a couple of the (large) commits which change the regression test artefacts, I can't be completely confident that all the changes are intentional. However, as long as the Content Team and/or the relevant Department are happy that the functionality is as required, I think this should be OK to merge.

**Comment by floehopper** [comment]
I expect you're already aware, but GitHub is indicating that there are conflicts between this branch and master which will need resolving before it can be merged.

**Comment by floehopper** [comment]
Other than my comments above, I'm happy for this to be merged.

File: lib/smart_answer_flows/am-i-getting-minimum-wage/outcomes/current_payment_below.govspeak.erb (hunk 1 above)
Comment (pmanrubia) [inline lib/smart_answer_flows/am-i-getting-minimum-wage/outcomes/current_payment_below.govspeak.erb]: Updated. Thanks

**Comment by pmanrubia** [comment]
Thanks for your review @floehopper, and for your comments on #2419. It might be interesting to write in our docs the approach that `smart-answers` is following when facing commits granularity (small vs squashed commits) cc/@chrisroos? It might help other devs to follow the same approach.

I have tackled all your comments, so I will rebase with master and force push in preparation for merging.

**Comment by pmanrubia** [comment]
Thanks for your review @floehopper, and for your comments on #2419. It might be interesting to write in our docs the approach that smart-answers is following when facing commits granularity (small vs squashed commits) cc/@chrisroos? It might help other devs to follow the same approach.

I have tackled all your comments, so I have rebased with master and force pushed in preparation for merging.

**Comment by floehopper** [comment]
I had already added [this Trello card][1] and I've just added [this other one][2] to write-up some documentation about the structure of commits.

[1]: https://trello.com/c/mSAVt8J0
[2]: https://trello.com/c/pigWkA6X

**Pull request merged**